webapp/src/Entity/Clarification.php

```diff
@@ -33,7 +33,8 @@
 #[UniqueEntity(fields: 'externalid')]
 class Clarification extends BaseApiEntity implements
     HasExternalIdInterface,
-    ExternalIdFromInternalIdInterface
+    ExternalIdFromInternalIdInterface,
+    PrefixedExternalIdInShadowModeInterface
```

Comment:
Maybe document for which case we get this locally? We always expect to read clarifications from the eventfeed but don't disallow clarifications to be sent from the local instance but are never expected to be used outside of the system. For clarifications we should even not put them in the eventfeed if they are local in case we need to swap systems in the future?

Reply:
Added a comment to the interface itself.